pkg/signup/service/signup_service_test.go

```diff
@@ -1158,6 +1160,9 @@ func (s *TestSignupServiceSuite) TestGetSignupStatusOK() {
 			require.Equal(t, us.Name, response.Name)
 			require.Equal(t, "jsmith", response.Username)
 			require.Equal(t, "ted", response.CompliantUsername)
+			require.NotNil(t, response.DaysRemaining)
+			require.Exactly(t)
+			require.Equal(t, float64(30), *response.DaysRemaining)
```

Comment:
the `StartDate` and `EndDate` should be set as well, right? Could you please verify it?

Reply:
Right, fixed.